### PR TITLE
Update A7b1 - Recording pre-solve penalties

### PR DESCRIPTION
**Pull request by CarterKoala**
No longer require penalties from before the solve (such as inspection penalties) to be listed before the time on the scorecard.

### CODE REVIEW

File: wca-regulations.md [hunk 1]
```diff
@@ -443,7 +443,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
...
     - A7b) The judge records the result on a score sheet and signs next to the result to acknowledge that the result is correct, complete, properly formatted, and clearly readable.
-        - A7b1) If penalties are assigned, the judge records the original recorded time displayed on the timer, along with any penalties. The format should be "X + T + Y = F", where X represents the sum of time penalties before/starting the solve, T represents the time displayed on the timer (the "original recorded time"), Y represents a sum of time penalties during/after the solve, and F represents the final result (e.g. 2 + 17.65 + 2 = 21.65). If X and/or Y is 0, the 0 terms are omitted (e.g. 17.65 + 2 = 19.65).
+        - A7b1) If penalties are assigned, the judge records the original recorded time displayed on the timer, along with any penalties. The format should be "T + X = F", where T represents the time displayed on the timer (the "original recorded time"), X represents a sum of time penalties before/during/after the solve, and F represents the final result (e.g. 17.65 + 4 = 21.65). If X is 0, the 0 term is omitted (e.g. 17.65 + 2 = 19.65).
```
Comment (CarterKoala) [inline wca-regulations.md]:
```suggestion
        - A7b1) If penalties are assigned, the judge records the original recorded time displayed on the timer, along with any penalties. The format should be "T + X = F", where T represents the time displayed on the timer (the "original recorded time"), X represents a sum of time penalties during the attempt, and F represents the final result (e.g. 17.65 + 4 = 21.65). If X is 0, the 0 term is omitted (e.g. 17.65 + 2 = 19.65).
```

**Comment by Nanush7** [comment]
LGTM!

**Comment by nsilvestri** [comment]
Looks like the final example still needs to be updated. Perhaps:

> If there are no penalties, only the original recorded time is required. (e.g. "17.65 + 0 = 17.65" should instead be written as simply "17.65").

File: wca-regulations.md [hunk 2]
```diff
@@ -443,7 +443,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
...
     - A7b) The judge records the result on a score sheet and signs next to the result to acknowledge that the result is correct, complete, properly formatted, and clearly readable.
-        - A7b1) If penalties are assigned, the judge records the original recorded time displayed on the timer, along with any penalties. The format should be "X + T + Y = F", where X represents the sum of time penalties before/starting the solve, T represents the time displayed on the timer (the "original recorded time"), Y represents a sum of time penalties during/after the solve, and F represents the final result (e.g. 2 + 17.65 + 2 = 21.65). If X and/or Y is 0, the 0 terms are omitted (e.g. 17.65 + 2 = 19.65).
+        - A7b1) If penalties are assigned, the judge records the original recorded time displayed on the timer, along with any penalties. The format should be "T + X = F", where T represents the time displayed on the timer (the "original recorded time"), X represents a sum of time penalties during the attempt, and F represents the final result (e.g. 17.65 + 4 = 21.65). If X is 0, the 0 term is omitted (e.g. 17.65 + 2 = 19.65).
```
Comment (CarterKoala) [inline wca-regulations.md]:
```suggestion
        - A7b1) If penalties are assigned, the judge records the original recorded time displayed on the timer, along with any penalties. The format should be "T + X = F", where T represents the time displayed on the timer (the "original recorded time"), X represents a sum of time penalties during the attempt, and F represents the final result (e.g. 17.65 + 4 = 21.65). If X is 0, the 0 term is omitted (e.g. 17.65).
```

**Comment by xsrvmy** [comment]
Actually I think the final example should be deleted. The reg is only applied if there is a penalty, and the example has no penalty.

File: wca-regulations.md [hunk 3]
```diff
@@ -443,7 +443,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
...
     - A7b) The judge records the result on a score sheet and signs next to the result to acknowledge that the result is correct, complete, properly formatted, and clearly readable.
-        - A7b1) If penalties are assigned, the judge records the original recorded time displayed on the timer, along with any penalties. The format should be "X + T + Y = F", where X represents the sum of time penalties before/starting the solve, T represents the time displayed on the timer (the "original recorded time"), Y represents a sum of time penalties during/after the solve, and F represents the final result (e.g. 2 + 17.65 + 2 = 21.65). If X and/or Y is 0, the 0 terms are omitted (e.g. 17.65 + 2 = 19.65).
+        - A7b1) If penalties are assigned, the judge records the original recorded time displayed on the timer, along with any penalties. The format should be "T + X = F", where T represents the time displayed on the timer (the "original recorded time"), X represents a sum of time penalties during the attempt, and F represents the final result (e.g. 17.65 + 4 = 21.65). If X is 0, the 0 term is omitted (e.g. 17.65).
```
Comment (CarterKoala) [inline wca-regulations.md]:
```suggestion
        - A7b1) If penalties are assigned, the judge records the original recorded time displayed on the timer, along with any penalties. The format should be "T + X = F", where T represents the time displayed on the timer (the "original recorded time"), X represents a sum of time penalties during the attempt, and F represents the final result (e.g. 17.65 + 4 = 21.65).
```
@xsrvmy You are definitely right about that - this case is already covered by A7b :)

**Review by Rouxles — APPROVED** [review]

**Pull request merged**